python/pyspark/errors/exceptions/captured.py

```diff
@@ -379,17 +386,12 @@ class UnknownException(CapturedException, BaseUnknownException):
     """
 
 
-class QueryContext(BaseQueryContext):
+class SQLQueryContext(BaseQueryContext):
```

Comment:
Do we consider this a private / developer API?

Reply:
Only parent class `QueryContext` is an API (at `pyspark.errors.QueryContext`) for now. This is at least consistent with Scala side.